sanic/src/server.py

```diff
@@ -3,9 +3,16 @@
 from sanic.request import Request
 from sanic_ext import openapi
 from pathlib import Path
+import numpy as np
 import feature_extraction as fe
 import tempfile
 import os
+import utils
+
+model = utils.load_model(os.path.join("models", "test")) #TODO fix whhen there is a model
+feature_extractor = utils.build_feature_extractor() #TODO Fjern hvis den ikke skal bruges
```

Comment:
Er det muligt at bruge en model flere gange concurrently? for ellers kan det blive et problem med flere requests